openpype/hosts/maya/api/lib.py

```diff
@@ -2629,12 +2629,17 @@ def load_capture_preset(data=None):
     options['viewport_options'] = temp_options
     options['viewport2_options'] = temp_options2
 
+    # CAMERA OPTIONS
+    id = 'Camera Options'
+    camera_options = {}
+    for key, value in preset[id].items():
```

Comment:
Admittedly - I totally agree, but I'm basically matching the exact code style that is used across that function. I feel like then it might just be better to refactor the whole thing.

Reply:
Actually, let me refactor that beast.

Reply:
Refactor who with the what now